benchmarks/Gemfile

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+# Keep updated to latest Ruby, and keep in sync with benchmarks/.ruby-version.
+ruby "2.7.1"
+
+gem "benchmark-ips"
+gem "memery"
+gem "memo_wise", git: "https://github.com/panorama-ed/memo_wise.git"
```

Comment:
Uh oh @JacobEvelyn shouldn’t this source the MemoWise from the relative directory locally? That is, wouldn’t we expect the benchmarks to run against the codebase version they are sitting in, including potentially uncommitted local changes?

Reply:
Ah, I forgot we could do that as we do with `outtake`!